packages/uiweb/src/lib/context/chatContext.ts

```diff
@@ -1,29 +1,49 @@
-import { Env } from "@pushprotocol/restapi";
+import { Env,IMessageIPFS, IUser } from "@pushprotocol/restapi";
 import { Constants } from "../config";
 import { createContext } from "react";
 
+
 export interface IChatDataContextValues {
     account: string | null;
     setAccount: React.Dispatch<React.SetStateAction<string| null>>;
     decryptedPgpPvtKey: string | null;
     setDecryptedPgpPvtKey: React.Dispatch<React.SetStateAction<string | null>>;
     env: Env;
     setEnv: React.Dispatch<React.SetStateAction<Env>>;
+    pushChatSocket: any;
+    setPushChatSocket: React.Dispatch<React.SetStateAction<any>>;
+    isPushChatSocketConnected: boolean;
+    setIsPushChatSocketConnected: React.Dispatch<React.SetStateAction<boolean>>;
+    connectedProfile: IUser | undefined;
+    setConnectedProfile: (connectedProfile: IUser) => void;
 }
 
 export const initialChatDataContextValues: IChatDataContextValues = {
-    account: '',
+    account: null,
     setAccount: () => {
       /**/
     },
-    decryptedPgpPvtKey: '',
+    decryptedPgpPvtKey: null,
```

Comment:
ig this has been changed in the theme one so just make sure it's changed to pgpPrivatekey after merging

Reply:
okay sure